File: findClosestPerson.go
func findClosest(x int, y int, z int) int {
	xDiff := int(math.Abs(float64(z - x)))
	yDiff := int(math.Abs(float64(z - y)))

	if xDiff < yDiff {
		return 1
	} else if yDiff < xDiff {
		return 2
	} else {
		return 3
	}
}

File: findClosestPerson.ts
function findClosest(x: number, y:number, z:number): number {
    let xDiff = Math.abs(z - x)
    let yDiff = Math.abs(z - y)

    if(xDiff < yDiff) {
        return 1
    }else if (yDiff < xDiff) {
        return 2
    }else{
        return 0
    }
}